Source: pullmoll
Upstream: No
Reason: Add wrapper because 2nd parameter for g_clear_pointer() is void (*func)(NotifyNotification* n)

--- plugins/housekeeping/csd-disk-space.c	2018-06-22 20:58:17.000000000 +0200
+++ plugins/housekeeping/csd-disk-space.c	2018-10-26 09:48:30.754831801 +0200
@@ -216,6 +216,12 @@
 }
 
 static void
+notify_close_callback (NotifyNotification *n)
+{
+	notify_notification_close (n, NULL);
+}
+
+static void
 nemo_empty_trash_cb (GObject *object,
                          GAsyncResult *res,
                          gpointer _unused)
@@ -796,7 +802,7 @@
         g_clear_object (&ldsm_monitor);
         g_clear_object (&settings);
         g_clear_object (&dialog);
-        g_clear_pointer (&notification, notify_notification_close);
+        g_clear_pointer (&notification, notify_close_callback);
         g_slist_free_full (ignore_paths, g_free);
         ignore_paths = NULL;
 }
